packages/modular-scripts/src/utils/gitActions.ts

```diff
@@ -54,13 +54,8 @@ function getModifiedAndUntrackedFileChanges(): string[] {
   return [];
 }
 
-// Get all diffed files from git remote origin HEAD
-function getGitDiff(targetBranch: string): string[] {
-  // get a commit sha between the HEAD of the current branch and git remote origin HEAD
-  const sha = execa.sync('git', ['merge-base', 'HEAD', targetBranch], {
-    cwd: getModularRoot(),
-  });
-
+// Get all diffed files in standardised order exluding unknown and removed files
+function runGitDiff(gitDiffCommand: string): string[] {
```

Comment:
Extracted this functionality out to its own function so it can be used from both `getGitDiff` and `getStagedFiles`.